### PR TITLE
merge stuff from n1trux/awesome-sysadmin (closes #127)

### LINKED ISSUE #127: Reduce overlap with awesome-sysadmin
Followup to https://github.com/Kickball/awesome-selfhosted/pull/81; our list currently overlaps with @n1trux's https://github.com/n1trux/awesome-sysadmin in many areas. To prevent overlap, avoid NIH syndrome and decrease maintenance on the 2 lists, I suggest merging some of our lists sections as follows:

Import and remove from awesome-sysadmin:
- [x] copy https://github.com/n1trux/awesome-sysadmin#project-management
  - [x] remove section from awesome-sysadmin
- [x] copy https://github.com/n1trux/awesome-sysadmin#statistics
  - [x] remove sectrion from awesome-sysadmin https://github.com/n1trux/awesome-sysadmin/pull/123
- [x] copy https://github.com/n1trux/awesome-sysadmin#ticketing-systems
  - [x] remove section from awesome-sysadmin https://github.com/n1trux/awesome-sysadmin/pull/122
- [x] copy https://github.com/n1trux/awesome-sysadmin#xmpp
  - [x] remove section from awesome-sysadmin - https://github.com/n1trux/awesome-sysadmin/pull/138
- [x] copy https://github.com/n1trux/awesome-sysadmin#webmail
  - [x] remove section from awesome-sysadmin
- [x] copy https://github.com/n1trux/awesome-sysadmin#wiki-software
  - [ ] split file storage and synchronization section in subsections
  - [x] remove section from awesome-sysadmin
- [x] copy https://github.com/n1trux/awesome-sysadmin#cloud-storage
  - [x] remove section from awesome-sysadmin
- [x] ~~copy https://github.com/n1trux/awesome-sysadmin#code-review~~
  - [x] ~~remove section from awesome-sysadmin~~
- [x] copy https://github.com/n1trux/awesome-sysadmin#collaborative-software
  - [x] remove section from awesome-sysadmin https://github.com/n1trux/awesome-sysadmin/pull/301
- [x] copy https://github.com/n1trux/awesome-sysadmin#continuous-integration--continuous-deployment to awesom-sysadmin https://github.com/n1trux/awesome-sysadmin/pull/302
  - [x] remove section from awesome-selfhosted https://github.com/Kickball/awesome-selfhosted/pull/1646

---

Delegate and link to awesome-sysadmin:
- [x] ~~link to https://github.com/n1trux/awesome-sysadmin#control-panels~~
- [x] link to https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
- [x] link to https://github.com/n1trux/awesome-sysadmin#dns
- [x] link to https://github.com/n1trux/awesome-sysadmin#identity-management
- [x] link to https://github.com/n1trux/awesome-sysadmin#it-asset-management
- [x] ~~link to https://github.com/n1trux/awesome-sysadmin#mail-servers~~
- [x] link to https://github.com/n1trux/awesome-sysadmin#monitoring - https://github.com/n1trux/awesome-sysadmin/pull/139 https://github.com/Kickball/awesome-selfhosted/pull/477
- [x] link to https://github.com/n1trux/awesome-sysadmin#web
- [x] ~~link to https://github.com/n1trux/awesome-sysadmin#version-control~~ let's keep it in a-selfhosted
- [ ] link to https://github.com/n1trux/awesome-sysadmin#status-pages
- [x] link to https://github.com/n1trux/awesome-sysadmin#vpn
- [x] ~~link to https://github.com/n1trux/awesome-sysadmin#rdbms~~ no existing section in selfhosted
- [x] send patches to awesome-sysadmin for software currently in these sections on awesome-selfhosted
- [x] remove above sections from awesome-selfhosted

---

Network and computer systems management software would live in awesome-sysadmin, while we keep communication systems and non-IT-related services and applications here. I'm still unsure about Mail Servers and Web Servers. Maybe this could be an exception and be present in both lists (they are important for both). Not sure about Analytics either.

@n1trux would this be ok? Everything here can be discussed

### PR DESCRIPTION
**Pull request by n1trux**
I didn't include Code Review, Collaborative Software and Continuous Integration as mentioned in #127.

If this is merged, I will delete the corresponding sections in awesome-sysadmin and will link to awesome-selfhosted as replacement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -404,6 +410,21 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [MaterialPoll](https://github.com/nearbycoder/materialpoll) - an opensource strawpoll.me clone for creating random polls. - ([Source code](https://github.com/nearbycoder/materialpoll), [Demo](http://materialpoll.tk))
   * [OpenSondage](https://github.com/leblanc-simon/OpenSondage) - an online service for planning an appointment or make a decision quickly and easily: Make a poll, Define dates or subjects to choose, Send the poll link to your friends or colleagues, Discuss and make a decision - ([Demo](http://framadate.org/), [Example](https://framadate.org/aqg259dth55iuhwm))
 
+## Project Management
```
Comment (nodiscc) [inline README.md]: can you move this to the existing Project Management subsection under Software Development?

File: README.md [hunk 2]
```diff
@@ -471,6 +492,30 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 See https://staticsitegenerators.net and https://www.staticgen.com
 
+
+## Statistics
```
Comment (nodiscc) [inline README.md]: Can you move this to the existing Analytics section?

File: README.md [hunk 3]
```diff
@@ -471,6 +492,30 @@ CMS are a practical way to setup a website with many features. CMS often come wi
...
+*Analytics software.*
+
+* [AWStats](http://www.awstats.org/) - Generates web, streaming, ftp or mail server statistics graphically.
+* [GoAccess](http://goaccess.io/) - Real-time web log analyzer and interactive viewer that runs in a terminal.
+* [Open Web Analytics](http://www.openwebanalytics.com/) - Add web analytics to websites using JS, PHP or REST APIs.
+* [Piwik](http://piwik.org/) - Web analytics application.
+* [Webalizer](http://www.webalizer.org/) - Fast web server log file analysis.
+
+
+## Ticketing systems
```
Comment (nodiscc) [inline README.md]: Can you move this to the existing Bug and Task Trackers subsection under Software Development?

File: README.md [hunk 4]
```diff
@@ -478,6 +523,15 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [YOURLS](http://yourls.org/) - YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener. Features include password protection, URL customization, bookmarklets, statistics, API, plugins, jsonp ([Source code](https://github.com/YOURLS/YOURLS/), [Documentation](https://github.com/YOURLS/YOURLS/wiki)) - `MIT`
 
 
+## Webmail
```
Comment (nodiscc) [inline README.md]: an you move this to the existing Email subsection under Communication Systems?

File: README.md [hunk 5]
```diff
@@ -500,6 +558,22 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [WiKiss](http://wikiss.tuxfamily.org/) - un wiki simple à utiliser et déployer.
   * [Wikitten](https://github.com/victorstanciu/Wikitten) - Wikitten is a small, fast, PHP wiki, and the perfect place to store your notes, code snippets, ideas, and so on.
 
+
+## XMPP
```
Comment (nodiscc) [inline README.md]: Can you move this to the existing XMPP subsection in Communication Systems?

**Comment by nodiscc** [comment]
Thanks for the quick patch @n1trux ; Some of the imported sections are misplaced (we already have sections for those, I pointed to them in the PR review above). If you could fix those points then we'd be happy to merge.

Thanks again, really appreciated

File: README.md (hunk 1 above)
Comment (n1trux) [inline README.md]: but project management isn't only about Software Dev., shouldn't we move it the other way round? :)

**Comment by n1trux** [comment]
ah, didn't see them, sorry. Some private emergency came up this minute, it may take a while to correct the rest :p

**Comment by n1trux** [comment]
@nodiscc could you look at this again? :)

I commented on project management and followed your directions for the rest.

File: README.md (hunk 1 above)
Comment (nodiscc) [inline README.md]: You are right, but then we would have to sort between dev and non-dev oriented software...

What can be done: move this to Software Development/Project Management subsection but leave the section header here, and just link to the other one:

``See [Software Development Platforms/Project Management](#project-management)``

File: README.md [hunk 6]
```diff
@@ -471,6 +509,20 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 See https://staticsitegenerators.net and https://www.staticgen.com
 
+
+## Ticketing systems
+*Web-based ticketing system.*
```
Comment (nodiscc) [inline README.md]: Could we move this to the existing Task and Bug trackers subsection? We might need to edit the wording (include Ticketing)

**Comment by nodiscc** [comment]
Thanks! We have to find a way to organize those 2 sections (see patch comments), and we'd be good to merge. Licenses and Languages can be added later.

File: README.md (hunk 1 above)
Comment (n1trux) [inline README.md]: I think we should differentiate between ticketing (as in sysadmin) and bug tracking (as in developer).

How about splitting the Bug and Task trackers category into bug trackers and "task trackers and ticketing systems", moving the latter to the top level and linking to it from software dev?

File: README.md (hunk 6 above)
Comment (n1trux) [inline README.md]: I think we should differentiate between ticketing (as in sysadmin) and bug tracking (as in developer).

How about splitting the Bug and Task trackers category into bug trackers and "task trackers and ticketing systems", moving the latter to the top level and linking to it from software dev?

Comment (nodiscc) [inline README.md]: Yes, this is even better.

 * Split section into
  - [ ] Software Dev/Bug Trackers
  - [ ] Ticketing
 - [ ] Link Software Dev/Bug Trackers -> Ticketing
 - [ ] Link Ticketing -> Software Dev/Bug Trackers

File: README.md (hunk 1 above)
Comment (n1trux) [inline README.md]: >You are right, but then we would have to sort between dev and non-dev oriented software...

Why not? ;)

I found out that the project management link in the ToC already links to the software dev / project management. Will integrate it.

**Comment by n1trux** [comment]
@nodiscc done, but I'm still not happy with how that Bug/Task Trackers list looks. It's long and mixes Bug Trackers and stuff like Kanban boards which are entirely different.

Split it in Bug Trackers and Task management and move the latter to top level? But I think you should merge first.

See also: https://github.com/jyguyomarch/awesome-productivity

**Comment by n1trux** [comment]
sorry to annoy you @nodiscc, but pinging to see if you still want to change something. I'd recommend you merge the stuff to avoid future merge conflicts :/

**Comment by nodiscc** [comment]
@n1trux thanks for the heads up; there are some minor things that can be fixed later. I'll work on the merge in the coming days.

**Comment by nodiscc** [comment]
Merged in #200; @n1trux @Kickball we might need to think a bit more about sections/organization

**Pull request closed without merging**